crates/parser2/src/parser/expr.rs

```diff
@@ -164,9 +169,19 @@ fn infix_binding_power<S: TokenStream>(parser: &mut Parser<S>) -> Option<(u8, u8
         Amp => (100, 101),
         LShift | RShift => (110, 111),
         Plus | Minus => (120, 121),
```

Comment:
Please add `is_aug` branch for aug assign. Also, you need to implement it for all binary operations. I think you can refactor the implementation so that you don't need to add if branches to all binary operators.